README.md

```diff
@@ -76,6 +76,8 @@ information on how to submit your changes to this repository.
 
 ## Sample view
 
+See ./examples for docker-compose file with Grafana if you'd like to quickly get a test environment up and running.
```

Comment:
I think it's worth keeping volume mapping in the compose file but drawing attention to it in the README should help, as not everyone will have their ceph confs in `/etc` depending on their setup. That link is indeed helpful but we don't need majority of the things it covers (as you have already added those) - just a link to docker compose along with a little explanation on setting `DOCKERHOST` variable will certainly help for new folks trying to get this up.